src/app/home/components/Card/Status/CardStatus.tsx

```diff
@@ -6,28 +6,23 @@ import { Grid, GridItem } from '@patternfly/react-core';
 
 const CardStatusComponent = ({ type, dataList, ...props }) => {
   let successList = [];
-  const failureList = [];
-  if (type === 'repositories') {
-    // hasReadyCondition = plan.MigPlan.status.conditions.filter(c => c.type === 'Ready').length > 0;
-    // hasErrorCondition =
-    //   plan.MigPlan.status.conditions.filter(c => c.category === 'Critical').length > 0;
+  let failureList = [];
+  let successfulNames = [];
 
-    successList = dataList.filter(item => {
-      if (item.MigStorage.status) {
-        return item.MigStorage.status.conditions.filter(c => c.type === 'Ready');
-      } else {
-        failureList.push(item);
-      }
-    });
+  if (type === 'repositories') {
+    successList = dataList.filter(item =>
+      item.MigStorage.status && item.MigStorage.status.conditions.filter(c => c.type === 'Ready').length !== 0
+      );
+    successfulNames = successList.map(item => item.MigStorage.metadata.name);
```

Comment:
We should move all of this logic to a selector. You are right about needing to refactor this, but while we are fixing it, we should move it out of the presentation layer & into a selector: https://github.com/fusor/mig-ui/blob/master/src/app/plan/duck/selectors.ts